kqueen_ui/blueprints/ui/templates/ui/partial/selectform.html

```diff
@@ -106,7 +118,20 @@
 
 {% macro render_field(field) %}
   <div class="form-group{% if field.errors %} has-error{% endif %}">
-    <label for="{{ field.id }}" class="control-label">{{ field.label }}</label>
+    <label for="{{ field.id }}" class="control-label">{{ field.label }}
+    {% if field.label.text == "Engine" %}
+      <i class="mdi mdi-help-circle"
+         data-toggle="popover"
```

Comment:
If we decide to put the link into a separate icon, then the regular info messages should be opened on hover. Such icons as help and info don't even look like something clickable